TestApp/TestApp/Model/Question.cs

```diff
@@ -44,10 +43,13 @@ public Question(int questionId, string questionType, string testQuestion, string
         public string IncorrectAnswer2 { get; set; }
         public string CourseName { get; set; }
         public int Point { get; set; }
-        public int Time { get; set; }
```

Comment:
removed the Time property since we aren't setting time per question. We set it on the test. Also there is no Time column in the database for question either